doc/deprecation_process.md

```diff
@@ -180,29 +204,21 @@ Check to make sure that the new version of the package has been released on PyPI
       - name: azure-mypackage
         safeName: azuremypackage
   ```
- - If the only package listed is `azure-mypackage`, remove the `Artifacts` section altogether.
-    ```yml
-    extends:
-      parameters:
-        ...
-    ```
- - If there are multiple packages listed, remove the `name` and corresponding `safeName` lines for only `azure-mypackage`.
+  - If there are multiple packages listed, remove the `name` and corresponding `safeName` lines for only `azure-mypackage`.
     ```yml
     extends:
       parameters:
         ...
         Artifacts:
           ...
     ```
+  - If the only package listed is `azure-mypackage`, remove the following files from `mypackage`: `ci.yml`, `tests.yml`, `test-resources.bicep`, `test-resources.json`, `perf.yml`, `perf-tests.yml`, `perf-resources.bicep`.
```

Comment:
Above we say remove all files except README, but then we list a specific set to remove here. Is it fair for a single package to say "remove everything except README", but for a multi-package directory give more detailed advice?

Reply:
discussed offline

Reply:
re-worded to make it less confusing!